servicetalk-http-router-jersey/src/main/java/io/servicetalk/http/router/jersey/EndpointEnhancingRequestFilter.java

```diff
@@ -496,13 +495,6 @@ private static HttpExecutionStrategy difference(final Executor fallback,
             return right;
         }
 
-        final Executor rightExecutor = right.executor();
```

Comment:
how does this relate to `HttpExecutionStrategies#difference`, and does this method still need to exist?

Reply:
`final Executor fallback,` parameter is no longer used

Reply:
It is now the same as the `HttpExecutionStrategies#difference` implementation

Reply:
Now removed.